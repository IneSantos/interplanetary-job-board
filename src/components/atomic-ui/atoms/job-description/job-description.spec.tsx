import React from "react";
import { render } from '@testing-library/react';
import JobDescription from "./job-description";

describe("JobDescription atom tests", () => {
  test(`JobDescription component render`, () => {
    const { container } = render(
      <JobDescription description="This is a description"/>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
