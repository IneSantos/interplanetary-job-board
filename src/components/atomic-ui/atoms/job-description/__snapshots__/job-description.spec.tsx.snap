// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JobDescription atom tests JobDescription component render 1`] = `
<div
  class="description"
>
  This is a description
</div>
`;
